website/pages/en/tutorials.js

```diff
@@ -286,7 +286,7 @@ class Tutorials extends React.Component {
                 difficulty: "easy",
                 length: "4",
                 prerequisite: false,
-                version: "2.0-3e65111",
+                version: "2.0.0-alpha.3",
```

Comment:
This tutorial isn't really versioned. See also https://github.com/substrate-developer-hub/substrate-contracts-workshop/issues/60

Up to you how to tag it here @riusricardo

Reply:
RIght, it should be working with the latest tag. Good recommendation. Doing it that way, we can keep the version tag.